app/components/DhbsVaccineDoseDataList/DhbsVaccineDoseDataList.module.scss

```
@@ -0,0 +1,22 @@
+@import "../../styles/spacing.scss";
+
+.grid {
```

Just something a little more specific could be good 🤷 

```suggestion
.card-grid {
```